automatic/lightalloy/update.ps1

```diff
@@ -0,0 +1,27 @@
+import-module au
+
+$releases = 'http://www.light-alloy.ru/download/'
+
+function global:au_SearchReplace {
+  @{
+    ".\tools\chocolateyInstall.ps1" = @{
+      "(?i)(^\s*packageName\s*=\s*)('.*')" = "`$1'$($Latest.PackageName)'"
+      "(?i)(^\s*url32\s*=\s*)('.*')" = "`$1'$($Latest.Url32)'"
+      "(?i)(^\s*checksum32\s*=\s*)('.*')" = "`$1'$($Latest.Checksum32)'"
+    }
+  }
+}
+
+function global:au_GetLatest {
+  $download_page = Invoke-WebRequest -Uri $releases
```

Comment:
Odd that it worked for you, when I tried running the update I got a vector smash, which means the package wont get updated.